ui/app/components/app-item/deployment.hbs

```diff
@@ -7,29 +7,66 @@
-      <:small-text>
-        {{#if (not-eq @latest.id @deployment.id)}}
-        <span class="replacement-info">
-          {{t "page.deployments.replaced_label"}}<b class="badge badge--version">v{{@latest.sequence}}</b>
+    <div class="app-item__deployment-state">
+      {{!-- ROW: status report + deployment url --}}
+      {{#if @deployment.statusReport.health}}
+        {{#let (hash
+          deploymentHealth=@deployment.statusReport.health.healthStatus
+          urlOrStatus=(or
+            (clean-up-url @deployment.preload.deployUrl)
+            (t (concat "status_report_indicator.label."
+              (downcase @deployment.statusReport.health.healthStatus))
+            )
+          )
+        ) as |vars|}}
+          <span
+            data-test-health-status={{downcase vars.deploymentHealth}}
+            class="deploy-url__status deploy-url__status--{{downcase vars.deploymentHealth}}"
+          />
+          <span class="deploy-url" title={{vars.urlOrStatus}}>
+            {{vars.urlOrStatus}}
+          </span>
+        {{/let}}
+      {{/if}}
+      {{!-- ROW: operation status --}}
+      {{#let @deployment.status as |status|}}
+        <FlightIcon
+          data-test-time-icon
+          @name={{or
+            (and (eq status.state 0) "help")
+            (and (eq status.state 1) "clock")
+            (and (eq status.state 2) "check")
+            (and (eq status.state 3) "alert-triangle")
+          }}
+          @size="16"
+        />
+        <span data-test-operation-status>
+          {{t (concat "page.deployments.status_prefix." status.state)}}
+          {{#if (eq status.state 1)}} {{!-- RUNNING --}}
+            {{t "page.deployments.started_time_prefix"}} {{date-format-distance-to-now status.startTime.seconds}}
+          {{else}}
+            {{date-format-distance-to-now status.completeTime.seconds}}
+          {{/if}}
         </span>
-        {{/if}}
-      </:small-text>
-      <:badge>
-        {{#if (eq @deployment.state 4)}}
-          <b data-test-destroyed-badge class="badge badge--destroyed">
-            <FlightIcon @name="trash" class="icon" />&nbsp; {{t "page.deployments.destroyed_label"}}
-          </b>
-        {{/if}}
-      </:badge>
-    </OperationStatusIndicator::Deployment>
+      {{/let}}
+      {{!-- ROW: git commit information --}}
+      {{#if @deployment.preload.jobDataSourceRef.git.commit}}
+        {{#let @deployment.preload.jobDataSourceRef.git.commit as |commit|}}
+          {{!-- TODO: change to if statement- if user profile picture available, use that. else use flight icon --}}
+          <FlightIcon @name="git-commit"/>
+          <span data-test-git-commit class="git-commit" title="Triggered by commit: {{commit}}">
+            <CopyButton @clipboardText={{commit}}>
+            <span class="git-commit__sha">{{truncate-commit commit}}</span>
+            </CopyButton>
+            {{!-- TODO: when commit message available, add to this --}}
```

Comment:
Following on from our discussion of TODOs the other day, I think it could be just a little clearer that we’re saying “*when* Waypoint gains the ability to store commit author and message data *then* add…”